src/main/java/eu/execom/pomodoroTeam/services/TeamService.java

```diff
@@ -46,4 +55,53 @@ public TeamEntity updateTeam(Long id, String name) {
         }
         return teamRepository.save(team);
     }
-}
+
+    public Invitation getByActivationLink(String activationLink) {
+        return invitationRepository.findOneByActivationLink(activationLink);
+    }
+
+    public NewInvitationDto DataFromInvitation(String activationLink) {
```

Comment:
Please adhere to Java standards -> method names should not start with an upper case. Also, it is a convention to add a verb to a method name, saying what the method does -> `getInvitationData`.